### PR TITLE
HL-288: add handler field and application handling status

### PR DESCRIPTION
**Pull request by haavikko**
## Description :sparkles:
* New ApplicationStatus.HANDLING logic
* Calculation.handler field + API for reading handler info

## Testing :alembic:
backend/benefit/calculator/tests/test_calculator_api.py
backend/benefit/applications/tests/test_applications_api.py

### CODE REVIEW

File: backend/benefit/calculator/api/v1/serializers.py [hunk 1]
```diff
@@ -27,7 +29,20 @@ class Meta:
...
+    handler_details = UserSerializer(
+        help_text="The handler object, with fields, currently assigned to this calculation and application (read-only)",
+        read_only=True,
+        source="handler",
+    )
+    handler = serializers.PrimaryKeyRelatedField(
+        required=False,
+        allow_null=True,
+        queryset=User.objects.filter(is_staff=True),
+        help_text="Set the handler of this Calculation (write-only)",
```
Comment (quyenlq) [inline backend/benefit/calculator/api/v1/serializers.py:45]: Can we add `write_only=True` here?

File: backend/benefit/applications/tests/test_applications_api.py [hunk 2]
```diff
@@ -22,7 +25,7 @@
 from applications.tests.conftest import *  # noqa
 from applications.tests.factories import ApplicationFactory, DecidedApplicationFactory
 from common.tests.conftest import *  # noqa
-from companies.tests.conftest import *  # noqa
+from companies.tests.conftest import *  # noqaa
```
Comment (quyenlq) [inline backend/benefit/applications/tests/test_applications_api.py:28]: typo

**Review by quyenlq — APPROVED** [review]
Left some nitpicking comments, but overall looks good to me 👌

File: backend/benefit/calculator/api/v1/serializers.py (hunk 1 above)
Comment (haavikko) [reply backend/benefit/calculator/api/v1/serializers.py:45]: Yes, good catch, I think I had it there at some point, wondering how it got lost

File: backend/benefit/applications/tests/test_applications_api.py (hunk 2 above)
Comment (haavikko) [reply backend/benefit/applications/tests/test_applications_api.py]: fixed

**Pull request merged**